### PR TITLE
added webpack dev build for other pages

### PR DESCRIPTION
**Pull request by PinkaminaDianePie**
mostly work for all pages, but some stuff still TBD:
- iframe pages doesn't work
- popup with source code for non-interactive pages
- search in the header

### CODE REVIEW

File: src/docs/layouts/IndexPage.jsx [hunk 1]
```diff
@@ -0,0 +1,181 @@
+// @flow
```
Comment (PinkaminaDianePie) [inline src/docs/layouts/IndexPage.jsx]: mostly copy-paste of original index, with some changes needed for SPA

File: src/docs/layouts/NavigationSPA.jsx [hunk 2]
```diff
@@ -0,0 +1,57 @@
+// @flow
```
Comment (PinkaminaDianePie) [inline src/docs/layouts/NavigationSPA.jsx]: mostly copy-paste of original navigation, but with react-router instead of normal links

File: .eslintrc [hunk 3]
```diff
@@ -11,7 +11,8 @@
     "./node_modules/frontend-tools-configs/eslint/flow.js"
   ],
   "rules": {
-    "max-len": ["error", { "code": 120, "ignoreComments": true }]
+    "max-len": ["error", { "code": 120, "ignoreComments": true }],
+    "react/no-multi-comp": "off"
```
Comment (PinkaminaDianePie) [inline .eslintrc]: our configs and eslint versions are so old, that bumping should be performed in a separate PR. in our latest configs i believe we don't have `react/no-multi-comp` rule anymore (or at least allow us to use arrows in JSX, so after bump of deps this could be removed)

File: src/docs/webpackEntry.jsx [hunk 4]
```diff
@@ -3,20 +3,82 @@ import React from 'react';
...
+const version = packageJSON.version;
 
 const requireAll = r => {
   r.keys().forEach(r);
 };
 
 requireAll(require.context('../images/', true, /\.svg$/));
 
-const Wrapped = hot(Entry);
+const GeneratePage = ({padeIndex}) => navigation[padeIndex].elements.map(({name, component}) => (
```
Comment (maciekbb) [inline src/docs/webpackEntry.jsx]: I'd split it into few lines :P

Comment (PinkaminaDianePie) [reply src/docs/webpackEntry.jsx]: done
